driftbase/auth/__init__.py

```diff
@@ -14,6 +14,11 @@
     'eos': 'driftbase.auth.eos',
     'ethereum': 'driftbase.auth.ethereum',
     'cognito': 'driftbase.auth.cognito',
+    'discord': 'driftbase.auth.discord',
+    # 'twitter': 'driftbase.auth.twitter',
```

Comment:
Please add a note about why these are disabled for the next reader. I'd suggest adding a 400 error or exception on trying to enable these too before they are fixed.

Reply:
Fixed in the latest